src/lib/divider/divider.md

```diff
@@ -0,0 +1,64 @@
+`<mat-divider>` is a container component that wraps and formats a series of line items. As the base
+list component, it provides Material Design styling, but no behavior of its own.
+
+<!-- example(divider-overview) -->
```

Comment:
There doesn't seem to be a divider example?

Reply:
Do you mean in the `material-examples` directory?

Reply:
I see now that the prose is wrong here, will correct.

Reply:
Yep. It would probably be good to add one in with this PR anyway.

Reply:
Added